Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.swift

```diff
@@ -58,8 +58,16 @@ import BraintreeCore
     /// If set to `.notRequested` the authentication challenge will not be requested from the issuer.
     /// If set to `.unspecified`, when the amount is 0, the authentication challenge will be requested from the issuer.
     /// If set to `.unspecified`, when the amount is greater than 0, the authentication challenge will not be requested from the issuer.
-    public var cardAddChallenge: BTThreeDSecureCardAddChallenge = .unspecified
-
+    @available(*, deprecated, message: "Use `cardAddChallengeRequested` instead")
```

Comment:
I think we can use:
```swift
@available(*, deprecated, renamed: "cardAddChallengeRequested" message: "whatever we want for the message here")
```
I believe this way lets you use `Fix` in Xcode.

Reply:
Ooo that's awesome, fixed!